geode-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisService.java

```diff
@@ -87,4 +88,9 @@ private void stopRedisServer() {
   public CacheServiceMBeanBase getMBean() {
     return null;
   }
+
+  @VisibleForTesting
+  public GeodeRedisServer getGeodeRedisServer() {
```

Comment:
if we remove that internal check mentioned above, we don't need this getter.

Reply:
Truth.